.github/workflows/ci.yaml

```diff
@@ -18,10 +18,8 @@ jobs:
       - run: make fmt
       - run: make tidy
       - run: |
-          sudo apt-get install -y software-properties-common
-          sudo add-apt-repository ppa:oisf/suricata-stable
           sudo apt-get update
-          sudo apt-get install -y suricata
+          sudo apt-get install -y zip libnet1 libjansson4
```

Comment:
Can't this step go away?

Reply:
No, the libraries are needed.

Reply:
Won't that be a problem for Brim? (We go out of our way to link Zeek statically for that reason.)

Reply:
As discussed elsewhere, I'd like use the regular .deb/.rpm dependency mechanism for this (just like the regular suricata .deb/.rpm does). If that doesn't work, then we can do the work to build statically. For now this does work for zq CI, which is what we need for the moment.